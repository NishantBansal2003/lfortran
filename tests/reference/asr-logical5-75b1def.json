{
    "basename": "asr-logical5-75b1def",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/logical5.f90",
    "infile_hash": "f44f503aa7262fed1bd2e925e83435b4d7bdcb63ea5833a3fc7a6113",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-logical5-75b1def.stdout",
    "stdout_hash": "dff35916bcace7c14d76caeab86d9a8175221c67d749126d6f0181d5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}