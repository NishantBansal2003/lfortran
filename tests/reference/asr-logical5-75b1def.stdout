(TranslationUnit
    (SymbolTable
        1
        {
            logical5:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b:
                                (Variable
                                    2
                                    b
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            c:
                                (Variable
                                    2
                                    c
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    logical5
                    []
                    [(Assignment
                        (Var 2 a)
                        (Cast
                            (LogicalConstant
                                .true.
                                (Logical 4)
                            )
                            LogicalToInteger
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 b)
                        (Cast
                            (LogicalConstant
                                .false.
                                (Logical 4)
                            )
                            LogicalToInteger
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 c)
                        (Cast
                            (LogicalConstant
                                .true.
                                (Logical 4)
                            )
                            LogicalToInteger
                            (Integer 4)
                            ()
                        )
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 a)
                            (Var 2 b)
                            (Var 2 c)]
                            FormatFortran
                            (Character -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
